docker/base-deps/README.md

```diff
@@ -4,19 +4,27 @@ This is an internal image, the [`rayproject/ray`](https://hub.docker.com/reposit
 
 This image  has the system-level dependencies for `Ray` and the `Ray Autoscaler`. The `ray-deps` image is built on top of this. This image is built periodically or when dependencies are added. [Find the Dockerfile here.](https://github.com/ray-project/ray/blob/master/docker/base-deps/Dockerfile)
 
+## Tags
 
+Images are `tagged` with the format `{Ray version}[-{Python version}][-{Platform}]`. `Ray version` tag can be one of the following:
 
+| Ray version tag | Description |
+| --------------- | ----------- |
+| `latest`                     | The most recent Ray release. |
+| `x.y.z`                      | A specific Ray release, e.g. 1.12.1 |
+| `nightly`                    | The most recent Ray development build (a recent commit from Github `master`) |
+| `6 character Git SHA prefix` | A specific development build (uses a SHA from the Github `master`, e.g. `8960af`). |
 
-## Tags
-* [`:latest`](https://hub.docker.com/repository/docker/rayproject/base-deps/tags?page=1&name=latest) - The most recent Ray release.
-* `:1.x.x` - A specific release build. 
-* [`:nightly`](https://hub.docker.com/repository/docker/rayproject/base-deps/tags?page=1&name=nightly) - The most recent nightly build.
-* `:DATE` - A specific build.
-
-### Suffixes
-* `-cuXXX` - These are based off of an `NVIDIA CUDA` image. They require the [Nvidia Docker Runtime](https://github.com/NVIDIA/nvidia-docker) to be installed on the host for the container to access GPUs.  
-* `-cpu`- These are based off of an `Ubuntu` image.
-* Tags without a suffix refer to `-cpu` images
+The optional `Python version` tag specifies the Python version in the image. All Python versions supported by Ray are available, e.g. `py37`, `py38`, `py39` and `py310`. If unspecified, the tag points to an image using `Python 3.7`.
+
+The optional `Platform` tag specifies the platform where the image is intended for:
+
+| Platform tag | Description |
+| --------------- | ----------- |
+| `-cpu`  | These are based off of an Ubuntu image. |
+| `-cuXX` | These are based off of an NVIDIA CUDA image with the specified CUDA version `xx`. They require the Nvidia Docker Runtime. |
+| `-gpu`  | Aliases to a specific `-cuXX` tagged image. |
+| no tag  | Aliases to `-cpu` tagged images for `ray`, and aliases to ``-gpu`` tagged images for `ray-ml`. |
```

Comment:
this is really nice. does add some example help here?

Reply:
Added examples for the `ray` and `ray-ml` images.